aidial_assistant/json_stream/json_node.py

```diff
@@ -59,11 +79,14 @@ async def to_string_chunks(self) -> AsyncIterator[str]:
 
     @staticmethod
     async def collect(stream: CharacterStream) -> str:
```

Comment:
`PrimitiveNode.collect` is an analog of `ReadableNode.read_to_end`. Maybe give it the same name then: `PrimitiveNode.read_to_end`.